cvmfs/cvmfs_suid_helper.cc

```diff
@@ -94,7 +110,8 @@ static void Mount(const string &path) {
       }
       systemd_unit = cvmfs_suid::EscapeSystemdUnit(resolved_path);
     }
-    ExecAsRoot("/bin/systemctl", "restart", systemd_unit.c_str(), NULL);
+    RunAsRoot("/bin/systemctl", "restart", systemd_unit.c_str(), NULL);
```

Comment:
Perhaps `ExecAsRootOrAbort`?

Reply:
I was trying to convey the non-`exec()` nature of the function `RunAsRoot()` (i.e. the fact that it actually returns, rather than replacing the current process).  Happy to switch to `ExecAsRootOrAbort()` if you prefer.

Reply:
I've switched to `ForkAndExecAsRoot()`: let me know if that's ok

Reply:
That makes more sense, thanks!